predicators/envs/pybullet_blocks.py

```diff
@@ -216,15 +216,24 @@ def _initialize_pybullet(self) -> None:
 
     def _create_pybullet_robot(
             self, physics_client_id: int) -> SingleArmPyBulletRobot:
-        ee_home = (self.robot_init_x, self.robot_init_y, self.robot_init_z)
+        robot_ee_orn = self._robot_ee_home_orn
+        ee_home = Pose(
+            (self.robot_init_x, self.robot_init_y, self.robot_init_z),
+            robot_ee_orn)
         return create_single_arm_pybullet_robot(CFG.pybullet_robot,
                                                 physics_client_id, ee_home)
 
     def _extract_robot_state(self, state: State) -> Array:
+        # The orientation is fixed in this environment.
+        q0, q1, q2, q3 = self._robot_ee_home_orn
```

Comment:
I think these are xyzw quaternions, so maybe better to call them `qx, qy, qz, qw`? Same comment throughout changes